extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ColumnPageReaderImpl.java

```diff
@@ -105,14 +105,16 @@ public class ColumnPageReaderImpl implements ColumnPageReader {
 
     @Override
     public Object materialize(Object nullValue) throws IOException {
-        try (final SeekableByteChannel readChannel = channelsProvider.getReadChannel(filePath)) {
+        try (final SeekableByteChannel readChannel =
```

Comment:
This context creation part will be changed after Ryan's patch and context will directly be passed from upper layers.